extensions/roulette/database/action.py

```diff
@@ -0,0 +1,23 @@
+from ..config import config
+from typing import Set
+
+
+class Role:
+    def __init__(self, id: int):
+        self.id = id
+
+
+def _get_timeout_roles(guild) -> Set[Role]:
+    role_ids_conf = config.timeout_roles()
+    role_ids = set()
+
+    for role_id_conf in role_ids_conf:
+        role = guild.get_role(int(role_id_conf))
+        if role:
```

Comment:
note: Could you add a TODO here that says "check if a call to `fetch_roles` is needed? I'm concerned there may be a scenario where the `guild` object doesn't actually contain the roles, so this doesn't do anything